hermes/bin/src/cli/mod.rs

```diff
@@ -21,19 +31,45 @@ pub(crate) struct Cli {
 }
 
 /// Hermes cli commands
-#[derive(Subcommand, Clone)]
+#[derive(Subcommand)]
 enum Commands {
     /// Package the app
-    Package,
+    Package(package::PackageCommand),
 }
 
 impl Cli {
     /// Execute cli commands of the hermes
-    pub(crate) fn exec(self) -> anyhow::Result<()> {
-        println!("{BUILD_INFO}");
+    pub(crate) fn exec(self) {
+        println!("{}{}", Emoji::new("ℹ️", ""), style(BUILD_INFO).yellow());
+
+        let mut errors = Errors::new();
+        let log_level = std::env::var(ENV_LOG_LEVEL)
+            .unwrap_or_default()
+            .parse()
+            .unwrap_or_default();
+
+        let log_config = LoggerConfigBuilder::default()
+            .log_level(log_level)
+            .with_thread(true)
+            .with_file(true)
+            .with_line_num(true)
+            .build();
+
+        logger::init(&log_config).unwrap_or_else(|err| errors.add_err(err));
+
         match self.command {
             None => run::Run::exec(),
-            Some(Commands::Package) => todo!(),
+            Some(Commands::Package(cmd)) => cmd.exec(),
```

Comment:
The command should be `hermes module ...`  `hermes package ...` should be for full application packages, not wasm modules.

Reply:
We should also stub the `hermes module sign` and `hermes module inspect` commands with their parameters in this PR, but those commands can just return and not do any actual work in this PR other than defining them.